requirements-dev.txt

```diff
@@ -1,7 +1,7 @@
 ninja
 flaky
 numpy
-git+https://github.com/PennyLaneAI/pennylane.git@master
+pennylane~=0.28
```

Comment:
This should also be back to PennyLane master?

Reply:
Yes. But just after you fork the release branch. Right?

Reply:
Nope. The development requirements are always mainline with PennyLane, as they arent packages in the wheel

Reply:
Just changed it back.